forte/data/ontology/top.py

```diff
@@ -1168,12 +1170,74 @@ def __init__(
         )
 
 
+class Payload(Entry):
+    def __init__(
+        self,
+        pack: PackType,
+        modality: str,
+        payload_idx: int,
+    ):
+        self.payload_idx = payload_idx
+        self.modality = modality
+        super().__init__(pack)
+        self.cache = []
```

Comment:
Are we planning multiple pieces of content here? Maybe we don't need a list of items? For example, one payload is associated with one piece of image.

Reply:
OK. I think it makes sense. In DataPack, we have class variables `self.payloads` that stores a collection of payloads of possibly different modalities.